docs/can-guides/experiment/technology/html.md

```diff
@@ -31,7 +31,7 @@ Let's say you want to create a page that counts clicks like the following: (_cli
 <style>
   my-counter button {margin-left: 15px;}
 </style>
-<script type="text/steal-module">
+<script id="counter-demo" type="text/steal-module">
```

Comment:
I don’t think we should have to add these IDs for the demos to work correctly. Can this be done dynamically?

Reply:
@chasenlehara it is fixed in https://github.com/canjs/bit-docs-html-canjs/pull/563 I'm closing this PR